ci/docker/builder/Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM golang:1.20.1-buster
+FROM ghcr.io/nordsecurity/nordvpn-linux/golang:debian-buster-20240201
 
 LABEL org.opencontainers.image.source=https://github.com/NordSecurity/nordvpn-linux
 
-ENV GOBIN=/usr/bin
-
 RUN dpkg --add-architecture i386; \
```

Comment:
`&&` should be used instead of `;` - because using `;` would not stop on error and try to execute following command which could depend on previous command and fail also, so, it's wasting time.